packages/squircle-element-react/src/StaticSquircle.tsx

```diff
@@ -32,8 +40,21 @@ export const StaticSquircle = ({
       height,
       cornerRadius,
       cornerSmoothing,
+      topLeftCornerRadius,
+      topRightCornerRadius,
+      bottomRightCornerRadius,
+      bottomLeftCornerRadius,
```

Comment:
Could you attach a screenshot of how this looks like with both `cornerRadius` and one of the corners having different values?